File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = '';
    
    for(let s of rsp){
        
        if(s === "2")
            answer += "0";
        else if(s === "5")
            answer += "2";
        else
            answer += "5";
        
    }
    
    
    return answer;
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        
        String[] str_arr = rsp.split("");
        StringBuffer sb = new StringBuffer("");
        
        for(var i = 0 ; i < str_arr.length ; i++){
            if(str_arr[i].equals("2"))
                sb.append("0");
            else if(str_arr[i].equals("0"))
                sb.append("5");
            else
                sb.append("2");
                
        }
        
        return sb.toString();
    }
}